docs/README.md

```diff
@@ -117,6 +115,15 @@ with the version number as the name.  For example, if you are publishing the
 charts/partners/acme/awesome/0.1.0/
 ```
 
+For Community charts only, partners should follow process above, submit the _OWNERS_ file PR towards `main` branch under your charts/community directory within your orgranization directory. This PR should be sent prior to sending any charts to the repo. For example, for a community chart named awsome for an organization named acme, you would send it here:
+
+charts/community/acme/awesome/OWNERS
```

Comment:
Format the path for better readability, same as `charts/partners/acme/awesome/0.1.0/` above:

```
charts/community/acme/awesome/OWNERS
```

Reply:
thanks